packages/cypress/src/utils/versions.ts

```diff
@@ -2,5 +2,6 @@ export const nxVersion = require('../../package.json').version;
 export const eslintPluginCypressVersion = '^2.10.3';
 export const typesNodeVersion = '16.11.7';
 export const cypressVersion = '^11.0.0';
+export const cypressViteDevServerVersion = '^2.2.1';
 export const cypressWebpackVersion = '^2.0.0';
-export const webpackHttpPluginVersion = '^5.5.0';
+export const httpWebpackPluginVersion = '^5.5.0';
```

Comment:
@jaysoo Sorry for bothering you but it is still named http instead of html.

Reply:
Thank for the review @Phillip9587 ! For what it's worth, it does not really matter what it's called. But I understand that it makes sense to give it a name that's more accurate, to avoid confusion.